service/controller/app/v1/resource/clients/resource.go

```diff
@@ -54,18 +54,14 @@ func New(config Config) (*Resource, error) {
 	if config.ImageRegistry == "" {
 		return nil, microerror.Maskf(invalidConfigError, "%T.ImageRegistry must not be empty", config)
 	}
-	if config.TillerNamespace == "" {
-		config.TillerNamespace = "giantswarm"
-	}
```

Comment:
We now use the in cluster flag to determine which tiller namespace should be used.